src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-sync-pipe-modal/rgw-multisite-sync-pipe-modal.component.spec.ts

```diff
@@ -7,10 +7,17 @@ import { PipesModule } from '~/app/shared/pipes/pipes.module';
 import { ReactiveFormsModule } from '@angular/forms';
 import { CommonModule } from '@angular/common';
 import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { of } from 'rxjs';
+import { RgwMultisiteService } from '~/app/shared/api/rgw-multisite.service';
+
+const multisiteServiceMock = {
```

Comment:
lets have a class here as thats wat close to real implementation pf this service.
```suggestion
class RgwMultisiteServiceMock  {

 createEditSyncPipe() {}
```